### ISSUE TITLE
ndcg score wrong and not matching sklearn. 

### ISSUE THREAD

**Issue by SimonCW** [karlhigley/ranking-metrics-torch#3]
Hi @karlhigley ,

while working on my own implementation of ndcg and checking it against sklearn's and yours, I noticed that your ndcg implementation produces different scores than sklearn. I added a simple unit test in this PR to demonstrate: https://github.com/karlhigley/ranking-metrics-torch/pull/2

I am new to hypothesis and couldn't check why all your tests pass nevertheless. Maybe there is no bug and I am missing something? 
The dcg scores are fine by the way (see first added test in PR). Looks to me like a problem with the normalizing gains but I don't understand the tie-aware implementation yet.

**Comment by karlhigley** [comment]
Resolved in [this commit](https://github.com/karlhigley/ranking-metrics-torch/commit/d84b5449698baad5d44da4f586f523fe31cdc5cd). The issue was passing an array of integers for the true values, so I added a check/conversion in that case.

**Comment by karlhigley** [comment]
(Try installing 0.2.3.)
